tapir/core/tapir_email_base.py

```diff
@@ -29,15 +37,19 @@ def get_all_emails(default: bool | None = True) -> list[Type[TapirEmailBase]]:
         return [mail for mail in TapirEmailBase.__subclasses__()]
 
 
-def mails_not_mandatory(default: bool | None = True) -> List[Tuple[str, str]]:
+def get_mails_not_mandatory(
```

Comment:
Could we merge all the `get_mails...()` functions into a single `get_mails(enabled_by_default: bool, mandatory: bool)`?